barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -809,6 +847,135 @@ void prove_output_all(const std::string& bytecodePath, const std::string& witnes
     vinfo("vk as fields written to: ", vkFieldsOutputPath);
 }
 
+/**
+ * @brief Creates a Honk proof for an ACIR circuit, outputs the proof and verification key in binary and 'field' format
+ *
+ * Communication:
+ * - Filesystem: The proof is written to the path specified by outputPath
+ *
+ * @param bytecodePath Path to the file containing the serialized circuit
+ * @param witnessPath Path to the file containing the serialized witness
+ * @param outputPath Directory into which we write the proof and verification key data
+ */
+template <IsUltraFlavor Flavor>
+void prove_honk_output_all(const std::string& bytecodePath,
```

Comment:
Rather than "all" will you use "vk" or something like this? Unclear at a glance what "all" means, and there's no real benefit to using a short function name here IMO.

Reply:
all means the vk in binary, vk as fields, proof in binary, proof as fields, so it's not exactly easily written in a function name